page_analyzer/utils/db.py

```diff
@@ -1,10 +1,12 @@
 import os
-from typing import Any, List
+from typing import Any, Dict, List
 
 import psycopg2
 import psycopg2.extras
 from dotenv import load_dotenv
 
+from .url import get_url_status_code
```

Comment:
it's a good practice to use full path imports
```suggestion
from paga_analyzer.utils.url import get_url_status_code
```

Reply:
tho we don't need it now